q2_types/per_sample_sequences/_format.py

```diff
@@ -9,19 +9,34 @@
 import skbio.io
 import yaml
 import qiime2.plugin.model as model
+import pandas as pd
 
 from ..plugin_setup import plugin
 
 
 class FastqManifestFormat(model.TextFileFormat):
```

Comment:
Can you also update the `FastqAbsolutePathManifestFormat` sniffer to perform these validations? Those formats will mostly validate the same way, with the exception of relative vs absolute paths.